File: 1-Iniciante/1064.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main(void)
{
    int i, pares = 0;
    float numero, soma = 0.0;

    for (i = 0; i < 6; ++i) {
        cin >> numero;
        if (numero > 0) {
            soma += numero;
            ++pares;
        }
    }

    cout << pares << " valores positivos" << endl;
    cout << fixed << setprecision(1) << soma / pares << endl;

    return 0;
}

File: 1-iniciante/1064.py
s = 0
q = 0
for i in range(6):
    n = float(input())
    if n > 0:
        q += 1
        s += n
if q != 0: m = s / q
else: m = 0
print(q, 'valores positivos')
print('{:.1f}'.format(m))
